package.json

```diff
@@ -10,46 +10,49 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "@react-native-community/async-storage": "^1.6.2",
-    "@react-native-community/eslint-config": "^0.0.5",
-    "axios": "^0.19.0",
-    "babel-plugin-root-import": "^6.4.1",
-    "eslint-plugin-prettier": "^3.1.1",
-    "i18n-js": "^3.3.0",
+    "@babel/core": "^7.9.0",
+    "@babel/plugin-proposal-optional-chaining": "^7.9.0",
+    "@babel/runtime": "^7.9.2",
+    "@react-native-community/async-storage": "^1.8.1",
+    "@react-native-community/eslint-config": "^1.0.0",
+    "@react-native-community/masked-view": "^0.1.7",
+    "@react-navigation/native": "^5.1.3",
+    "@react-navigation/stack": "^5.2.8",
+    "@reduxjs/toolkit": "^1.3.2",
+    "axios": "^0.19.2",
+    "babel-jest": "^25.2.4",
+    "babel-plugin-module-resolver": "^4.0.0",
+    "babel-plugin-root-import": "^6.5.0",
+    "eslint": "^6.8.0",
+    "eslint-plugin-prettier": "^3.1.2",
+    "i18n-js": "^3.5.1",
+    "jest": "^25.2.4",
     "lodash": "^4.17.15",
-    "metro-react-native-babel-preset": "^0.56.0",
-    "prettier": "^1.18.2",
+    "metro-react-native-babel-preset": "^0.59.0",
+    "prettier": "^2.0.2",
     "prop-types": "^15.7.2",
-    "react": "16.9.0",
-    "react-native": "0.61.2",
-    "react-native-eject": "^0.1.1",
-    "react-native-gesture-handler": "^1.3.0",
-    "react-native-localize": "^1.1.4",
-    "react-native-paper": "^2.16.0",
-    "react-native-restart": "^0.0.13",
+    "react": "^16.11.0",
+    "react-native": "0.62.0",
+    "react-native-eject": "^0.1.2",
+    "react-native-gesture-handler": "^1.6.1",
+    "react-native-localize": "^1.3.4",
+    "react-native-paper": "^3.6.0",
+    "react-native-reanimated": "^1.7.1",
+    "react-native-restart": "^0.0.14",
+    "react-native-safe-area-context": "^0.7.3",
+    "react-native-screens": "^2.4.0",
     "react-native-vector-icons": "^6.6.0",
-    "react-navigation": "^4.0.10",
-    "react-navigation-hooks": "^1.1.0",
-    "react-navigation-stack": "^1.9.4",
-    "react-redux": "^7.1.1",
-    "redux": "^4.0.4",
+    "react-redux": "^7.2.0",
+    "redux": "^4.0.5",
     "redux-actions": "^2.6.5",
     "redux-logger": "^3.0.6",
     "redux-persist": "^6.0.0",
-    "redux-saga": "^1.1.1",
-    "redux-starter-kit": "^0.7.0",
-    "uuid": "^3.3.2"
+    "redux-saga": "^1.1.3",
+    "uuid": "^7.0.2"
   },
   "devDependencies": {
-    "@babel/core": "^7.6.3",
-    "@babel/plugin-proposal-optional-chaining": "^7.6.0",
-    "@babel/runtime": "^7.6.3",
-    "babel-jest": "^24.9.0",
-    "babel-plugin-module-resolver": "^3.2.0",
     "babel-plugin-transform-remove-console": "^6.9.4",
-    "eslint": "^6.5.1",
```

Comment:
The ESLint is updated by you but why you move the dev packages to the `dependency` part?

Reply:
my fault, now it's in dev